src/System.Text.Json/src/System/Text/Json/Node/DuplicatePropertyNameHandling.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace System.Text.Json
+{
+    /// <summary>
+    /// Specifies how duplicate property names are handled when added to JSON object.
+    /// </summary>
+    public enum DuplicatePropertyNameHandling
+    {
+        /// <summary>
+        /// Replace the existing value when there is a duplicate property. The value of the last property in the JSON object will be used.
+        /// </summary>
+        Replace,
+        /// <summary>
+        /// Ignore the new value when there is a duplicate property. The value of the first property in the JSON object will be used.
+        /// </summary>
+        Ignore,
+        /// <summary>
+        /// Throw an <exception cref="ArgumentException"/> when a duplicate property is encountered.
+        /// </summary>
+        Error
```

Comment:
Personal preference: The last value in an enum should end in a comma, so that if we ever add another member it leaves this line unedited.

It's the difference of

```diff
        Ignore,
        /// <summary>
        /// Throw an <exception cref="ArgumentException"/> when a duplicate property is encountered.
        /// </summary>
-       Error
+       Error,
+       SomeNewValue,
```

and

```diff
        Ignore,
        /// <summary>
        /// Throw an <exception cref="ArgumentException"/> when a duplicate property is encountered.
        /// </summary>
        Error,
+       SomeNewValue,
```

The latter diff is cleaner.